### PR TITLE
Add Everscale banner

### PR DESCRIPTION
**Pull request by babowsk1**

### CODE REVIEW

File: .build/website/src/pages/index.module.css [hunk 1]
```diff
@@ -3,16 +3,15 @@
...
   padding: 4rem 0;
   text-align: center;
   position: relative;
   overflow: hidden;
   background-repeat: no-repeat;
   background-position: top right;
-  /*sample for image*/
-  /*background-image: -webkit-image-set(url(https://everscale.network/images/Backgrounds/Main/main-hero.png) 1x, url(https://everscale.network/images/Backgrounds/Main/main-hero@2x.png) 2x);*/
-  /*background-image: image-set(url(https://everscale.network/images/Backgrounds/Main/main-hero.png) 1x, url(https://everscale.network/images/Backgrounds/Main/main-hero@2x.png) 2x);*/
+  background-image: -webkit-image-set(url(https://everscale.network/images/1x.png) 1x, url(https://everscale.network/images/2x.png) 2x);
```
Comment (ilyar) [inline .build/website/src/pages/index.module.css:13]: https://everscale-incubator.github.io/docs/PR-308/
image

**Review by ilyar — CHANGES_REQUESTED** [review]

File: .build/website/src/pages/index.module.css (hunk 1 above)
Comment (babowsk1) [reply .build/website/src/pages/index.module.css:13]: Can you suggest how to fix this?

Comment (babowsk1) [reply .build/website/src/pages/index.module.css:13]: Can you suggest how to fix this?

Comment (babowsk1) [reply .build/website/src/pages/index.module.css:13]: > https://everscale-incubator.github.io/docs/PR-308/
> image
> 

Can you suggest how to fix this?

Comment (ilyar) [reply .build/website/src/pages/index.module.css:13]: @babowsk1 https://hexlet.io/programs/frontend or https://djinni.co/developers/ or  https://github.com/everscale-org/bounties/issues/new/choose

Comment (ilyar) [reply .build/website/src/pages/index.module.css:13]: @babowsk1 or postpone the clearance decision and change the focus to higher priority documentation issues

**Comment by cryshado** [comment]
no activity

**Pull request closed without merging**